src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseHybridFVM.cpp

```diff
@@ -556,53 +495,42 @@ bool CompositionalMultiphaseHybridFVM::checkSystemSolution( DomainPartition cons
 {
   GEOSX_MARK_FUNCTION;
 
-  // 1. check cell-centered variables for each region
   string const elemDofKey = dofManager.getKey( viewKeyStruct::elemDofFieldString() );
-  localIndex localCheck = 1;
+  integer localCheck = 1;
```

Comment:
IIUC in `XXX::SolutionCheckKernel` the value for new pressure is checked to be non-negative ? Could add that as a comment to know what is checked.

Reply:
Yes, you are right, I put the comments that you suggested before all the kernel launches.